Makefile

```diff
@@ -95,8 +95,16 @@ release: all deps docs $(SMF_MANIFESTS)
 	$(TOP)/smf \
 	$(TOP)/npm \
 	$(RELSTAGEDIR)/$(NAME)
+	rm -rf $(RELSTAGEDIR)/$(NAME)/node_modules/eslint \
```

Comment:
Thinking out loud: could this be handled by specifying these deps as `devDepencies` in `package.json` and using `npm install --production` in the target `all` instead?